difficult_stage_1/main.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style3.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body onload='loadPage()'>
+    </script>
+    <h1 class = 'white' style='font-size: 50px;'>Lets guess</h1> 
+    <div class = 'key-input'>
+      <div class = 'key-input-group d-block text-center'>
+        <div id='qn'class = 'white' style='font-size: 20px;'></div>
+        <br/>
+        <div>
+          <div id='audio'>
+          </div>
+        </div>
+        <br/>
+        <label for='ans'>Answer:</label>
+        <input text='Ans' class='letter' id='input' name='input' required />
+      </div>
+    </div>
+    <br/>
+    <div class="text-center">
+      <button type="submit" id='submit' class='submit'>SUBMIT</button>
+    </div>
+    
+    <script src='script.js'>
+    </script>
+  </body>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script>
+    const audioSequence = [{qn:"which country was Alan Turing born in?", audioClip:"source/qn1 (1).mp3"}, {qn:"When did Bill Gates make the first computer?", audioClip:"source/qn2 (1).mp3"}, {qn:"What is the first mobile App in the world?", audioClip:"source/qn3 (1).mp3"},{qn:"What is the most popular computer language now?", audioClip:"source/qn4 (1).mp3"}, {qn:"What were Steve Jobs last designs?", audioClip:"source/qn5 (1).mp3"}]
+    let index = 0
+
+    function loadPage(){
+
+      //display audio
+      audioDisplay()
+      //display qn
+      qn = document.getElementById('qn')
+      qn.textContent = audioSequence[index].qn
+
+      console.log(index)
+    }
+
+    function audioDisplay() {
+      container = document.getElementById('audio')
+      var x = document.createElement('AUDIO')
+      x.canPlayType = 'audio/mpeg'
+      x.setAttribute('src', audioSequence[index].audioClip)
+      x.setAttribute('controls', 'controls')
+      container.appendChild(x)
+    }
+
+    function sent() {
```

Comment:
Correct me if I am wrong but I can't find where this function is used.

Reply:
it will be used when u click the submit button where it will move to the nxt question (addEventListener part)

Reply:
Ah. I see ok. 

Take note: I have commented on discord 2 ways to implement the POST request to the server, sorry for the miscommunication a while back but it turns out jquery `$.post{}` is not able to store cookies. Hence, one would have to use either `jquery Ajax` or `fetch` or even `XMLHttpRequest` (I didnt provide example for the last way)